core/src/gl/texture.cpp

```diff
@@ -155,18 +155,16 @@ void Texture::update(GLuint _textureUnit) {
 
         m_shouldResize = false;
 
-        m_subData.clear();
+    }
 
```

Comment:
Yeah, wasn't sure about what, isn't clearing subdata makes us loss information? We should probably fix the stride instead right?